### ISSUE TITLE
disconnection event

### ISSUE THREAD

**Issue by misterwolf** [websocket-rails/websocket-rails#357]
Hello,
when the page is reloaded, the client disconnection is triggered. 
is it possible avoid it?

Thanks

Dario

**Comment by phlegx** [comment]
No! When the page unloads to request another page (or browser/tab close), also the javascripts (websocket client) are unloaded. You can do a reconnect on page ready if you want:

```js
dispatcher.bind('connection_closed', function() {
  dispatcher.reconnect();
});
```

**Comment by darioAtIubenda** [comment]
Hello phlegx,
my apologies but I am not understanding a thing:
if I reload page, the dispatcher variable will no longer and the reconnection will not be launched because it is under the connection_close bind event.

There is an example online, so I can comprehend better?

Thank you!
